examples/filesystem/main.c

```diff
@@ -287,7 +287,7 @@ static int _tee(int argc, char **argv)
     }
     fclose(f);
 #else
-    int fd = open(argv[1], O_RDWR | O_CREAT);
+    int fd = open(argv[1], O_RDWR | O_CREAT, 00777);
```

Comment:
This might need some testing on embedded boards. I don't think that file system permissions received much testing and that this might expose some bugs. It would be a pitty, if we break the filesystem example by exposing bugs.

Reply:
that part of the example is only used on non new/picolibc devices (see l279), the previous call definitely was against the documentation of open my manpage states it document the call for:
```
CONFORMING TO
       open(), creat() SVr4, 4.3BSD, POSIX.1-2001, POSIX.1-2008.
```
the least significant 3 octal digits are also specified according to posix (even in their value) and i think 00777 is much easier to read  than  S_IRWXU.

if riot has other libc than posix compatible or new/picolibc a third case is needed since this is the only posix compliant variant create forces mode parameter to be set.